packages/strapi-admin/admin/src/containers/HomePage/index.js

```diff
@@ -132,13 +136,18 @@ export class HomePage extends React.PureComponent {
   };
 
   showFirstBlock = () =>
-    get(this.props.plugins.toJS(), 'content-manager.leftMenuSections.0.links', []).length === 0;
+    get(
+      this.props.plugins.toJS(),
+      'content-manager.leftMenuSections.0.links',
+      [],
+    ).length === 0;
 
   renderButton = () => {
     const data = this.showFirstBlock()
       ? {
         className: styles.homePageTutorialButton,
-        href: 'https://strapi.io/documentation/3.x.x/getting-started/quick-start.html#_3-create-a-content-type',
+        href:
+            'https://strapi.io/documentation/3.x.x/getting-started/quick-start.html#_3-create-a-content-type',
```

Comment:
We should use this link `https://strapi.io/documentation/getting-started/quick-start.html#_3-create-a-content-type` instead. It will redirect the user to the latest version available.